### PR TITLE
Adding Discount Test Script

### PR DESCRIPTION
**Pull request by stinkyfi**

### CODE REVIEW

**Comment by stinkyfi** [comment]
Adding testing script to confirm, discount functionality

**Comment by 54657566656c** [comment]
I would appreciate a README.md rework. Not needed for this PR, but as a future upgrade.

**Comment by stinkyfi** [comment]
Removed debug states per request

**Review by 54657566656c — APPROVED** [review]

File: test/DwarfDiscount.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    it("Verify Supply equals 0", async function () {
+      let supply = await DWARF.totalSupply();
+      expect(supply).to.be.equal(0);
+
+    });
+
+    it("Add Discounts to 2 Addresses, and mint", async function () {
+        await DWARF.addDiscounts([owner.address, addr1.address], [ethers.utils.parseEther("0.2"), ethers.utils.parseEther("0.5")]);
+
```
[the remark this answers is not in the document]
Comment (stinkyfi) [reply test/DwarfDiscount.js:57]: Helpful debug lines, useful for testing. 
Will remove for the merge into main

**Pull request merged**